Jenkinsfile

```diff
@@ -9,8 +9,8 @@ spec:
   securityContext:
     runAsUser: 1000 # default UID of jenkins user to node user in agent image
   containers:
-  - name: node14
-    image: 'node:14.17.6'
+  - name: node18
+    image: 'node:18.16.0'
```

Comment:
Can you use `node:18` or `node:18.16` instead, so you get security fixes?

Reply:
@bootswithdefer Done. Great suggestion.